src/scss/skin-modern/components/_button.scss

```diff
@@ -7,18 +7,31 @@
   background-position: center;
   background-repeat: no-repeat;
   border: 0;
+  // sass-lint:disable no-important
+  box-sizing: content-box !important;
```

Comment:
The 'global' box-sizing selector 
```scss
  *,
  *::before,
  *::after {
    box-sizing: border-box;
  }
```

in skin.scss has precedence so over this selector so important is needed here. A possible solution would be to move the box-sizing (in skin.scss) outside of the `.#{$prefix}-ui-uicontainer` but this would affect the styling on the entire page where the UI is included. Bootstrap does it too but i don't think it is a perfect solution.

What do you think @protyposis?

Reply:
We need to get rid of the `!important`, but let's postpone this for later. Maybe we get an idea in the meantime on how to solve this.

Reply:
Solved in https://github.com/bitmovin/bitmovin-player-ui/pull/181